app/uk/gov/hmrc/economiccrimelevyreturns/forms/mappings/Formatters.scala

```diff
@@ -24,6 +24,18 @@ import scala.util.control.Exception.nonFatalCatch
 
 trait Formatters {
 
+  private lazy val twoDecimalPattern = """^-?(\d*\.[0-9]{0,2})$""".r
```

Comment:
Are we allowing negative numbers here?

Reply:
The pattern passes with only a period and no decimal numbers

e.g. **1.**

Reply:
Removing it